### PR TITLE
feat: Add scopes to /login endpoint (no-changelog)

### PR DESCRIPTION
**Pull request by valya**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: packages/cli/src/services/user.service.ts [hunk 1]
```diff
@@ -120,6 +120,7 @@ export class UserService {
 
 		let publicUser: PublicUser = {
 			...rest,
+			globalScopes: user.globalScopes,
```
Comment (krynble) [inline packages/cli/src/services/user.service.ts:123]: Should we add a test to this? To make sure we're always returning this property at least.

**Review by krynble — DISMISSED** [review]
LGTM, left a suggestion.

File: packages/cli/src/services/user.service.ts (hunk 1 above)
Comment (valya) [reply packages/cli/src/services/user.service.ts]: I've added a quick check to the auth tests. These may need expanded on in the future though.

**Review by krynble — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.18.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.18.0)

**Pull request merged**